modules/dxp/apps/segments/segments-experiment-web/src/main/java/com/liferay/segments/experiment/web/internal/product/navigation/control/menu/SegmentsExperimentProductNavigationControlMenuEntry.java

```diff
@@ -172,9 +177,14 @@ public boolean includeIcon(
 		return true;
 	}
 
-	public boolean isPanelStateOpen(HttpServletRequest httpServletRequest) {
+	@Override
+	public boolean isPanelStateOpen(
+		HttpServletRequest httpServletRequest, String key) {
+
 		String segmentsExperimentPanelState = SessionClicks.get(
-			httpServletRequest, _SESSION_CLICKS_KEY, "closed");
+			httpServletRequest,
```

Comment:
@RenanUmV Please delegate this part https://github.com/liferay-core-infra/liferay-portal/pull/4599/commits/c814acde4168ac59aba431bd8535392aaf808d17#diff-bfe4e2b1785ce7e685688a7ceb5f83cdea5d6e05714d19d1aa3b923378dddb8cR184-R191 using `super.isPanelStateOpen`:

```
		if (super.isPanelStateOpen(httpServletRequest, key)) {
			return true;
		}
```